assets/js/custom/CloseEditorDialog.js

```diff
@@ -0,0 +1,26 @@
+import Swal from 'sweetalert2'
+
+export class CloseEditorDialog {
+  constructor (closeEditorPrompt, btnDiscardText, cancelText) {
+    this.closeEditorPrompt = closeEditorPrompt
+    this.btnDiscardText = btnDiscardText
+    this.cancelText = cancelText
+  }
+
+  show (confrimedCallback) {
+    Swal.fire({
+      title: this.closeEditorPrompt,
+      icon: 'question',
+      showCancelButton: true,
+      showCloseButton: true,
+      allowOutsideClick: true,
+      backdrop: true,
+      confirmButtonText: this.btnDiscardText,
+      cancelButtonText: this.cancelText
```

Comment:
I think the conventional options are "save" and "discard"

Reply:
Like instead of showing discard and cancel it should show save and discard?

Again, I am not sure if we should have two save buttons. I can remove the cancel button and then it will show discard and the user can close the popup to go back to the editor if you think that would be better though

Reply:
i think the 'save' 'discard' dialog is pretty standard is ui

Reply:
Okay, updated to save and discard. I ended up merging the two different popups since they had the same parameters